src/Classes/Providers/WindowsFeatureParcelProvider.ps1

```diff
@@ -8,17 +8,17 @@ class WindowsFeatureParcelProvider : ParcelProvider
             throw 'Windows Features are only supported on Windows'
         }
 
-        if ((Get-Host).Version.Major -gt '5') {
-            throw "Windows Features is only supported on PS5.0"
-        }
+        # if ((Get-Host).Version.Major -gt '5') {
+        #     throw "Windows Features is only supported on PS5.0"
+        # }
 
         return $true
     }
 
     [string] GetPackageInstallScript([ParcelPackage]$_package)
     {
         if ($this.IsOptionalFeature($_package)) {
-            return "Enable-WindowsOptionalFeature -FeatureName $($_package.Name) -NoRestart -All -Online -ErrorAction Stop"
+            return "DISM /Online /Enable-Feature /All /FeatureName:$($_package.Name) /NoRestart -ErrorAction Stop"
         }
 
         return "Add-WindowsFeature -Name $($_package.Name) -IncludeAllSubFeature -IncludeManagementTools -ErrorAction Stop"
```

Comment:
Is it possible to use DISM for this as well? Or does the above DISM command cover both Optional and Server Features?

Reply:
I think one command to rule them all.